docs/user-guide/plugins/custom-email-notifier.md

```diff
@@ -0,0 +1,61 @@
+# Custom Email Notifier
+
+## Introduction
+The **Custom Email Notifier** plugin by Devtron enables users to set automated email notifications to respective stakeholders of that specific CI/CD workflow. The Custom Email Notifier can be integrated into pre/post stages of Devtron CI/CD workflows to inform team members about the current status and other pipeline events. Some key use cases of this plugin can be:
+- Configuring notifications at pre/post stages to update teams on task completion status, such as sending vulnerability scanning reports.
+- Setting up automated alerts at the runbooks of auto-remediation using jobs.
+
+### Prerequisites
+Before integrating the **Custom Email Notifier** plugin, you must ensure that you have properly configured notifications in your Devtron setup. For detailed notification configuration steps, please refer to the Devtron documentation at: [Devtron Notifications](https://docs.devtron.ai/global-configurations/manage-notification).
+
+---
+
+## Steps
+
+{% hint style="warning" %}
+If you want to integrate the **Custom Email Notifier** plugin into pre/post stages of CI/CD pipelines, navigate to the **[Custom Webhook Notifier](https://docs.devtron.ai/usage/plugins/plugin-list/custom-webhook-notifier)** plugin and follow the steps mentioned.
```

Comment:
What's the relation of custom email notifier with webhook notifier?

Reply:
I understood now but this is confusing, mentioning it can be added as a task in job or ci/cd pipeline is fine